use_cases/common/single_type/metrics/two_series_arithmetic.yaml

```diff
@@ -0,0 +1,71 @@
+title: Two Series Arithmetic
+
+description: >
+  A common operation operation involves combining two or more time series into a single
+  aggregate. Used most often in systems that require a single explicit metric name 
+  during query time with optional filters on tags. 
+
+inputs:
+  - metric series
+  -
+uses:
+  - type: Alerting
+    description: >
+      Used for simple, single threshold alerting on a combination of series. 
+
+  - type: Visualization
+    description: >
+      Used to reduce the amount of data displayed in a visualization and combine
+      complementary data.
```

Comment:
I would say averages and ratios are more common than reducing amount of data displayed. For example average latency or percentage of failed requests.